src/index.js

```diff
@@ -0,0 +1,222 @@
+  F: [80, 70, 60, 50],
+  C: [27, 21, 15, 10],
+};
+
+window.onload = () => {
+  tempValue.textContent = findLatitudeAndLongitude();
+  handleTempandLandscapesChange();
+};
+
+const upAndDownButtons = () => {
+  const increaseButton = document.getElementById('increase-temp-control');
+  const decreaseButton = document.getElementById('decrease-temp-control');
+  increaseButton.addEventListener('click', () => {
+    tempValue.textContent = parseInt(tempValue.textContent) + 1;
+    handleTempandLandscapesChange();
+  });
+  decreaseButton.addEventListener('click', () => {
+    tempValue.textContent = parseInt(tempValue.textContent) - 1;
+    handleTempandLandscapesChange();
+  });
+};
+
+const cityContent = () => {
+  cityName.textContent = state.city;
+};
+
+///Wave 3\\\
+const updateCityName = () => {
+  cityName.textContent = state.city;
+  textboxCity.addEventListener('keyup', () => {
+    cityName.textContent = textboxCity.value;
+    state.city = textboxCity.value;
+  });
+};
+
+///Wave 4\\\
+const getRealTimeTemp = () => {
+  const realTimeTempButton = document.getElementById('current-temp-button');
+  realTimeTempButton.addEventListener('click', findLatitudeAndLongitude);
+};
+
+///Wave 5\\\
+const displaySky = () => {
+  const skyValue = document.getElementById('sky-select');
+  handleSkyChange();
+  skyValue.addEventListener('change', () => {
+    document.getElementById('sky-select');
+    handleSkyChange();
+  });
+};
+
+///Wave 6\\\
+const reset = () => {
+  const resetButton = document.createElement('button');
+  const resetContainer = document.getElementById('textbox-content');
+  resetButton.textContent = 'Reset';
+  resetContainer.appendChild(resetButton);
+  resetButton.addEventListener('click', () => {
+    state.city = 'Seattle';
+    textboxCity.value = state.city;
+    cityName.textContent = 'Seattle';
+    tempValue.textContent = findLatitudeAndLongitude();
+    handleTempandLandscapesChange();
+  });
+};
+
+///Wave 7\\\
+const celsius = () => {
+  const celsiusButton = document.getElementById('celsius-button');
+  celsiusButton.addEventListener('click', convertFToC);
+};
+
+const fahrenheit = () => {
+  const fahrenheitButton = document.getElementById('fahrenheit-button');
+  fahrenheitButton.addEventListener('click', convertCToF);
+};
+
+//Helper Functions\\
+const convertKToF = (temp) => {
+  return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const convertCToF = () => {
+  if (state.degree === 'C') {
+    tempValue.textContent = Math.round(tempValue.textContent * (9 / 5) + 32);
+    state.degree = 'F';
+  }
+};
+
+const convertFToC = () => {
+  if (state.degree === 'F') {
+    tempValue.textContent = Math.round((tempValue.textContent - 32) * (5 / 9));
+    state.degree = 'C';
+  }
+};
+
+const handleTempandLandscapesChange = () => {
+  const currentDegree = tempConversion[state.degree];
+  const currentTemp = parseInt(tempValue.textContent);
+
+  if (currentTemp >= currentDegree[0]) {
+    tempValue.style.color = 'red';
```

Comment:
Style: It would be better if you set a class here (like `hot`) so that this could live with the rest of your styles.

https://en.wikipedia.org/wiki/Separation_of_concerns